code/modules/projectiles/guns/energy/special.dm

```diff
@@ -195,6 +213,13 @@
 	ammo_type = list(/obj/item/ammo_casing/energy/bolt/large)
 	pin = null
 
+/obj/item/weapon/gun/energy/kinetic_accelerator/suicide_act(mob/user)
+	if(!suppressed)
+		playsound(src.loc, 'sound/weapons/kenetic_reload.ogg', 60, 1)
+	user.visible_message("<span class='suicide'>[user] cocks the [src.name] and pretends to blow \his brains out! It looks like \he's trying to commit suicide!</b></span>")

```

Comment:
if you have a suggestion of better flavor text that works for a KA *and* an e-bow, please let me know what it is (it can't be damage-oriented because ebows do tox)